quartz_solar_forecast/forecast.py

```diff
@@ -28,7 +26,7 @@ def run_forecast(site: PVSite, ts: datetime | str) -> pd.DataFrame:
         ts = datetime.fromisoformat(ts)
 
```

Comment:
I wounder if source should be a variable we put in to `run_forecast`, this means it can be passed into `get_nwp` and into `NwpDataSource`.
It can also be put into ` model.set_data_sources`

Reply:
makes sense

> The 'GFS' can be set to anything and it still works. Probably just make sense to set to nwp_source i.e ICON or GFS

I can do this but will need to add a `source` argument to `run_forecast`. Should I go ahead and add that argument?

Reply:
yea that would be super, and maybe default is `ICON`

perhaps it should be `nwp_source` to make it really clear what the source is about